website/static/overview.html

```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
 
 <div class="content homecontent width clearfix">
     <div class="">
-        <h1>Introduction</h1>
+        <h1>Introduction to Presto</h1>
 
         <p>
-            Presto is a distributed system that runs on a cluster of machines.
-            A full installation includes a coordinator and multiple workers.
+            Presto is a distributed system that runs on a cluster of machines. It enables 
+	analytics on large amounts of data. With Presto, access and query data in place on
+	many data different data sources (see image below). Presto supports ANSI SQL.
+        </p>
+
+   <!--suppress CheckImageSize -- larger file for retina -->
+        <img src="static/presto-overview-2.png" width="80%" alt="Presto System Overview">
+
+	<p>
+         A full installation includes a coordinator and multiple workers.
```

Comment:
This still says installation?

Reply:
hmmm, weird, this was supposed to be have all the commits but clearly its not working sorry - will follow up shortly.